sunkit_instruments/lyra/tests/test_lyra.py

```diff
@@ -58,7 +59,7 @@ def test_split_series_using_lytaf():
     dummy_data = np.random.random(seconds)
 
     lytaf_tmp = lyra.get_lytaf_events('2010-06-13 02:00', '2010-06-13 06:00',
-                                      combine_files=["ppt"])
+                                      combine_files=["ppt"], force_use_local_lytaf=True)
```

Comment:
This change should be a separate PR directly to master as it doesn't deal with IRIS stuff.